src/tabs.less

```diff
@@ -0,0 +1,37 @@
+.tabs {
+    .content {
+        display: none;
+        padding: 0.75rem 0 0;
+    }
+
+    input {
+        display: none;
+    }
+
+    label {
+        display: inline-block;
+        margin: 0 0 -1px;
+        padding: 0.75rem 0.75rem;
+        font-weight: 600;
+        text-align: center;
+        color: @primary-light;
+    }
+
+    label:hover {
+        color: @muted;
+        cursor: pointer;
+    }
+
+    input:checked+label {
+        color: @primary;
+        border-bottom: solid 3px @secondary;
+    }
+
+    #tab1:checked~#content1,
```

Comment:
should this css be here, it seems like this is very much linked to your example and wouldn't be reusable for someone using the framework

Reply:
Thanks for pointing out

Reply:
@Fraham I've added loop to select tabs id from 1 to 5. Can you help me get this id dynamically from HTML or should we arbitrarily set this limit up 10 or 20 tabs?